talk/morelanguage/lambda.tex

```diff
@@ -192,6 +192,33 @@
   \end{block}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Capture list}
+  \begin{block}{Capturing this}
+    Inside a (non-static) member function, we can capture \mintinline{cpp}{this}.
+  \end{block}
+  \begin{block}{}
+    \begin{cppcode*}{gobble=2}
+      [this](...) { use(*this); };
+      [&](...) { use(*this); };
+      [&, this](...) { use(*this); };
+      [=](...) { use(*this); };       // deprecated in C++20
+      [=, this](...) { use(*this); }; // allowed in C++20
+    \end{cppcode*}
+    The object at \mintinline{cpp}{*this} is captured by reference.
```

Comment:
That might be a bit confusing. This might not fit on the slide, but how about:
```suggestion
    \mintinline{cpp}{this} is a pointer. Therefore, referring to \mintinline{cpp}{*this} is like referring to the object by reference.
```

Reply:
I tried to elaborate further.

Reply:
That's good. Merge this if it fits on the slide. 🙂